wolfssl/openssl/evp.h

```diff
@@ -280,6 +297,7 @@ WOLFSSL_API int  wolfSSL_EVP_Cipher(WOLFSSL_EVP_CIPHER_CTX* ctx,
                           unsigned char* dst, unsigned char* src,
                           unsigned int len);
 
+WOLFSSL_API const WOLFSSL_EVP_CIPHER* wolfSSL_EVP_get_cipherbynid(int);
```

Comment:
Yes. My only comment on it is to not have the function take a nid argument and take care of setting and testing NIDs in the scope of the function. Also be careful to guard with OPENSSL_EXTRA and other macros expected for tests being ran i.e. checking that NO_AES is not defined when expecting AES NIDs to not return NULL.